health/api/src/main/java/com/cerner/beadledom/health/dto/BuildDto.java

```diff
@@ -41,22 +44,31 @@ public static Builder builder() {
   public abstract Optional<String> getBuildDateTime();
 
   @AutoValue.Builder
+  @JsonPOJOBuilder(withPrefix = "set")
   public abstract static class Builder {
 
+    @JsonCreator
+    private static BuildDto.Builder create() {
+      return BuildDto.builder();
+    }
+
     abstract Builder setArtifactName(Optional<String> artifactName);
 
+    @JsonProperty("artifactName")
```

Comment:
why do we need to give the property name explicitly. I thought Jackson automatically infers from the method name.

Reply:
The Beadledom default object mapper uses underscores, but the health check needs to use camel case.